### DISCUSSION TITLE
Login screen for hoobs hangs

### DISCUSSION THREAD

**Discussion by skyleranelson** [balansse/homebridge-vivint#93] [Q&A] [answered]
So configuring this plugin acts like it's working, but once you login it thinks, then just goes back to the login screen. If you manually put the credentials in it acts like it works, but just hangs on boot. Any ideas?

**Comment by jgrimard** [comment, marked as answer]
I would suggest posting in the Hoobs repo.  They use their own login screen and code.
[https://github.com/hoobs-org/sidecars/tree/main/hoobs-vivint-sidecar](https://github.com/hoobs-org/sidecars/tree/main/hoobs-vivint-sidecar)
